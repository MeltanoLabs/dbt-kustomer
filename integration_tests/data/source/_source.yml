version: 2

seeds:
  - name: customers
    config:
      column_types:
        type: text
        id: text
        attributes: jsonb        
        relationships: jsonb
        links: jsonb
        updated_at: timestamp
