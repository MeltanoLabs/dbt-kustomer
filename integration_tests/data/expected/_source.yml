version: 2

seeds:
  - name: stg_kustomer__customers_expected
    config:
      column_types:
        customer_id: text
        external_id: text
        name: text
        first_name: text
        last_name: text
        display_name: text
        gender: text
        default_language: text
        locale: text
        display_color: text
        progressive_status: text
        is_deleted: text
        is_verified: text
        conversation_count_open: text
        conversation_count_done: text
        conversation_count_snoozed: text
        conversation_count_all: text
        last_conversation_channels: text
        last_conversation_tags: text
        last_message_sent_at: text
        last_activity_at: text
        last_message_at: text
        modified_by: text
        created_by: text
        display_icon: text
        external_ids: text
        shared_external_ids: text
        emails: text
        shared_emails: text
        phones: text
        shared_phones: text
        whatsapps: text
        shared_whatsapps: text
        facebook_ids: text
        instagram_ids: text
        socials: text
        shared_socials: text
        urls: text
        locations: text
        active_users: text
        watchers: text
        recent_location: text
        recent_location_updated_at: text
        locale: text
        time_zone: text
        birthday_at: text
        presence: text
        created_at: text
        updated_at: text
        modified_at: text
        last_seen_at: text
        last_customer_activity_at: text
        deleted: text
        last_message_in_channel: text
        last_message_in_sentiment: text
        last_message_in_sent_at: text
        last_message_out_sent_at: text
        last_message_unresponded_to_sent_at: text
        last_message_unresponded_to_channel: text
        last_conversation_id: text
        last_conversation_channels: text
        last_conversation_tags: text
        last_conversation_sentiment: text
        conversation_counts_open: text
        conversation_counts_done: text
        conversation_counts_snoozed: text
        conversation_counts_all: text
        preview_channel: text
        preview_subject: text
        preview_text: text
        preview_type: text
        preview_preview_at: text
        tags: text
        sentiment_polarity: text
        sentiment_confidence: text
        custom: text
        verified: text
        rev: text
        recent_items: text
        satisfaction_level_count: text
        first_satisfaction_conversation_id: text
        first_satisfaction_sent_by_teams: text
        first_satisfaction_form_response_id: text
        first_satisfaction_form_id: text
        first_satisfaction_channel: text
        first_satisfaction_status: text
        first_satisfaction_scheduled_for: text
        first_satisfaction_created_at: text
        first_satisfaction_sent_at: text
        first_satisfaction_updated_at: text
        first_satisfaction_sent_by: text
        last_satisfaction_conversation_id: text
        last_satisfaction_sent_by_teams: text
        last_satisfaction_form_response_id: text
        last_satisfaction_form_id: text
        last_satisfaction_channel: text
        last_satisfaction_status: text
        last_satisfaction_scheduled_for: text
        last_satisfaction_created_at: text
        last_satisfaction_sent_at: text
        last_satisfaction_updated_at: text
        last_satisfaction_sent_by: text
        role_group_versions: text
        record_updated_at: timestamp
  
